src/blocks/video.js

```diff
@@ -9,11 +9,11 @@ module.exports = Block.extend({
   // more providers at https://gist.github.com/jeffling/a9629ae28e076785a14f
   providers: {
     vimeo: {
-      regex: /(?:http[s]?:\/\/)?(?:www.)?vimeo.com\/(.+)/,
+      regex: /(?:http[s]?:\/\/)?(?:www.)?vimeo\.co(?:.+(?:\/)([^\/].*)+$)/,
```

Comment:
Oh I wish it were so...for some bizarre reason (that's probably quite simple for regex masters who scoff at my mortal flailings) it wouldn't acknowledge the backslash at the end of the 'm' (I think it's because it was seeing all of ```vimeo.com\``` not ```vimeo.com```) so as a hack I nixed the m and roberts your mothers brother, the blasted thing works with vimeos various video ID methods rather than just a direct link where there's only the ID in the url path and nothing else.

And yes, I did try it without the slash (and a few other methods too) but this was the only one that worked with all variations of the video locations.